ga4gh/datamodel/datasets.py

```diff
@@ -6,25 +6,26 @@
 from __future__ import unicode_literals
 
 import fnmatch
-import json
 import os
 
 import ga4gh.datamodel as datamodel
 import ga4gh.datamodel.reads as reads
 import ga4gh.datamodel.sequenceAnnotations as sequenceAnnotations
 import ga4gh.datamodel.variants as variants
+import ga4gh.datamodel.ontologies as ontologies
 import ga4gh.exceptions as exceptions
 import ga4gh.protocol as protocol
 
 
-class AbstractDataset(datamodel.DatamodelObject):
+class Dataset(datamodel.DatamodelObject):
```

Comment:
I think FileSystemDataset should be removed, and the SimulatedDataset doesn't do anything that a regular Dataset does, other than create a bunch of objects that it contains. Therefore, I think we should get rid of this polymorphism and just use plan Dataset everywhere, and decide what it contains at run-time.

Having said that, I do regret changing the name right now. It has led to a lot of diff noise!